src/components/pacbio/PacbioPoolLibraryEdit.vue

```diff
@@ -146,18 +163,35 @@ export default {
     ...mapActions(['applyTags']),
     //return any errors exist in library for the given attribute
     errorsFor(attribute) {
+      this.setValidationRequired()
       return this.library?.errors?.[attribute]
     },
     attributeValueExists(attribute) {
+      this.setValidationRequired()
       return this.library?.[attribute]?.length > 0
     },
     isValidationExists(attribute) {
-      return this.attributeValueExists(attribute) || this.errorsFor(attribute)?.length > 0
+      if (this.validated) {
+        return true
+      } else {
+        return !this.validationRequiredFields[attribute]
+      }
+    },
+    setValidationRequired() {
+      if (this.validated) {
+        this.validationRequiredFields = []
+      }
+    },
```

Comment:
Do you think the method name is bit confusing? It is actually resetting all flags related to fields , probably something like resetAllValidationFields ?